### DISCUSSION TITLE
Which group of companies produces both cars and military aircraft?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#368] [Galaxy Trivia] [answered]
**Hint:** 💎💎💎

**Comment by Roberto-Larivera** [comment]
Saab AB

**Reply by ParanoidUser** [reply]
Good hit! However, Saab ceased operations in 2016 and this one is still functional. More ideas? 😊

**Comment by Roberto-Larivera** [comment]
Aermacchi?

**Reply by ParanoidUser** [reply]
Interesting, I heard that Aermacchi made motorcycles, but not sure if they ever made a car. 🤔 Anyway, they went out of business in 2003. How about global Japanese manufacturers?

**Comment by thuongtruong1009** [comment]
One group of companies that produces both cars and military aircraft is the Italian company, Fiat Chrysler Automobiles (FCA) and its subsidiary, Fiat Avio. FCA produces a range of automobiles under brands such as Fiat, Jeep, Dodge, and Chrysler, while Fiat Avio designs and manufactures components for aircraft engines, including military aircraft. Another example is the American company General Motors, which produces automobiles under brands such as Chevrolet and Cadillac, and also owns the subsidiary General Motors Defense, which produces military vehicles and equipment.

**Comment by ParanoidUser** [comment]
Hey mateys! Any idea which Japanese car manufacturer this might be?
@awesomelewis2007, @JustGimli, @JulioG516, @GerardoJuarezS, @brunoalksouza

**Reply by GerardoJuarezS** [reply]
I think China still has Good manufacturers.

**Comment by ParanoidUser** [comment]
**Hint 2:** Translated from Japanese, the company's name literally means three diamonds. The same symbol appears in their logo.

**Reply by Roberto-Larivera** [reply]
Kawasaki? Sabaru? 😊🙄

**Comment by Roberto-Larivera** [comment]
Kawasaki? Sabaru? 😊🙄

**Reply by ParanoidUser** [reply]
So close! 😊 Don't give up!

**Comment by 0x4248** [comment]
Honda is a car company in Japan but also make aircraft's.

**Reply by ParanoidUser** [reply]
Nice try! But, how does Honda's name/logo relate to x3 diamonds? 😊 Any other ideas?

**Comment by ParanoidUser** [comment]
**Hint 3:**
![logo](https://user-images.githubusercontent.com/5120290/227229999-c071ca68-6083-418f-bed4-e4a7693a75ae.png)

**Comment by Roberto-Larivera** [comment, marked as answer]
Mitsubishi

**Reply by ParanoidUser** [reply]
Bravo! 👏👏

**Reply by Roberto-Larivera** [reply]
https://it.m.wikipedia.org/wiki/Mitsubishi_Aircraft_Corporation
Mitsubishi Aircraft Comporation, che fa parte del gruppo Mitsubishi

**Reply by Roberto-Larivera** [reply]
@ParanoidUser Ciao non so perché non ricevo i badge Galaxy Brain da qui

**Reply by ParanoidUser** [reply]
Il prossimo `x2 Galaxy Brain` (Bronzo) viene assegnato per *8 risposte* accettate. Penso che ora ne hai 7 in totale, ne manca ancora uno. 😊

**Reply by Roberto-Larivera** [reply]
Grazie per l'informazione non ho ancora capito dove trovare delle istruzioni dettagliate di questi trofei badge
